app/components/template-viewer/template-viewer.sass

```diff
@@ -1,3 +1,23 @@
 #template-viewer
-  background: #D0D5EB
+  background: #FF757E
+  color: white
+  text-shadow: 1px 1px black
   overflow: hidden
+  display: block
+  border-radius: 5px
+
+.template-head
+    font-family: Helvettica
+    margin-bottom: 20px
+
+.template-item
+    border-style: solid
+    border-width: thin
+    border-color: black
```

Comment:
prefer sass variable over magic color (this may seem annoying but it has big benefits for styling down the road). variables are good.